ui/app/actions.js

```diff
@@ -502,7 +502,7 @@ function requestRevealSeed (password) {
     return new Promise((resolve, reject) => {
       background.submitPassword(password, err => {
         if (err) {
-          dispatch(actions.displayWarning(err.message))
+          dispatch(actions.displayWarning(err))
```

Comment:
@fvictorio this is because `submitPassword` actually returns string and not `Error` object:
https://github.com/vbaranov/KeyringController/blob/a3251d6306f155c23eba785ddf9ef75a2a609e32/index.js#L449. Buit it will be unified in the next version of `eth-keychain-controller`: all catches will return `Error` object.